### DISCUSSION TITLE
TF1 & TMC Channels

### DISCUSSION THREAD

**Discussion by NansMs** [iptv-org/community#291] [Q&A] [answered]
Hello,
Does anybody know if there's a reason the TF1 group French channels (TF1, TFX, TMC) are not present in the playlists ?
Thank you

**Comment by BellezaEmporium** [comment, marked as answer]
Basically 
-> Link changes all the time, is DRMd
-> To get the stream, it's geo-blocked, our services do not target specific countries for those hosts.

We used to have a stream that got the old TF1 CDN streams but they've changed it just before the World Cup came in, in October 2022.
